native/goldilocks_extension.cuh

```diff
@@ -91,6 +91,18 @@ struct __align__(16) extension_field {
     }
     return result;
   }
+
+  static DEVICE_FORCEINLINE extension_field shr(const extension_field &x, const unsigned &shift) {
```

Comment:
out of curiosity, do we actually use these anywhere?

Reply:
> out of curiosity, do we actually use these anywhere?

we do not, i just implemented it to make the API "complete"